sdcm/sla/sla_tests.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+import logging
+import time
+import traceback
+
+from sdcm.sct_events import Severity
+from sdcm.sct_events.workload_prioritization import WorkloadPrioritizationTestEvent
+from sdcm.sla.libs.sla_utils import SlaUtils
+from sdcm.utils.loader_utils import DEFAULT_USER, DEFAULT_USER_PASSWORD
+from test_lib.sla import create_sla_auth
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Steps(SlaUtils):
```

Comment:
The steps concept here is a bit problematic, since the catch the exception and rise event, the actual nemesis wouldn't fail...

it's o.k. for a functional test, that we want to keep on going on case of error in one step, but for nemesis I think we want to fail fast, or at least collect the errors and report them as part of the nemesis failure.

Reply:
changed